test/cpp/jit/test_class_type.cpp

```diff
@@ -93,13 +93,21 @@ TEST(ClassTypeTest, IdenticalTypesDifferentCus) {
   } catch (const std::exception& e) {
     // Check that the exception contains the address of the compilation unit
     // in addition to the ClassType's name.
+#ifdef _MSC_VER
+    testing::FileCheck()
```

Comment:
Referring to https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/class_type.h#L104, I am not sure what is the intent, whether to print the pointer, or the actual value and what is the return type on Linux/MacOS? It would be nicer to unify this "intent" with Linux/MacOS but if that's not possible, I'd rather `#ifdef` a constant to prevent test code duplication.

Reply:
It looks like the intent is to print the pointer, but I wonder if it would be more efficient to put ifdef there and add "0x" on Windows

Reply:
@Blackhex @malfet I just made some changes, was that what you had in mind?

Reply:
@everton1984 @malfet In the case printing the pointer has its semantical value, have you tried adding `std::showbase` to  https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/class_type.h#L104 as suggested in https://stackoverflow.com/questions/66802137/printing-addresses-with-0x-notation? If that change is not working as desired or if it is not feasible, I have no further comments to the current state of the PR.

Reply:
@Blackhex Thanks for the comments. I tried the `std::showbase` fix and it didn't work, that's the reason I ended up doing it like that.

Reply:
@malfet is this solution @everton1984 made ok for you?